velox/duckdb/conversion/DuckWrapper.cpp

```diff
@@ -204,7 +204,7 @@ VectorPtr convert(
       if (child.GetType() == LogicalTypeId::HUGEINT ||
           child.GetType() == LogicalTypeId::TIMESTAMP ||
           child.GetType() == LogicalTypeId::VARCHAR) {
-        std::vector<uint8_t> validityVector((maxIndex + 7) / 8, 0);
+        std::vector<uint8_t> validityVector(bits::nbytes(maxIndex + 1), 0);
```

Comment:
I don't think the `+ 1` is needed? 
```
inline uint64_t nbytes(int32_t bits) {
  return roundUp(bits, 8) / 8;
}
```

Reply:
I ran into the case when maxIndex was 0. In this case validityVector was of size 0 causing a crash.